packages/core-parse/src/extensions/spans.js

```diff
@@ -0,0 +1,50 @@
+/** Roles for wrapping inline text in containers with semantic meaning. */
+import { u } from 'unist-builder'
+
+import { RoleProcessor } from '../roleProcessor.js'
+
+export const spansExtension = {
+    name: 'spans',
+    /** @type {Record<string, {processor: typeof RoleProcessor}>} */
+    roles: {},
+}
+
+/** Taken from https://github.com/live-clones/docutils/blob/48bb76093b4ba83654b2f2c86e7c52c4bb39c63b/docutils/docutils/parsers/rst/roles.py#L257-L264 */
+for (const role of [
+    'abbreviation',
+    'acronym',
+    'literal',
+    'emphasis',
+    'strong',
+    'subscript',
+    'superscript',
+    'title-reference',
```

Comment:
What is this, can we remove?

Reply:
https://docutils.sourceforge.io/docs/ref/rst/roles.html#title-reference

Reply:
I feel like in the same way that we and not bringing everything over (e.g. pep-reference), this is probably better handled by a `cite` role.

Reply:
removed 👍